components/header.js

```diff
@@ -1,34 +1,72 @@
-import Link from "next/link"
+import Link from "next/link";
+import styles from "./style.module.css";
+import Image from "next/image";
+
+import React, { useEffect, useState } from "react";
 
 export default function Header() {
-    
-  
-    return (
-      <header>
-        <nav>
-          <ul className="">
-            <li className="">
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
```

Comment:
I miss some loading state to indicate the user the Async process

Reply:
i don't know how put a loading message while async function work

Reply:
ok, lets start to understand async in javascript I will catch up with you. in a meeting

Reply:
yes, i understand for what we use async function in javascript. The problem is how i do some asyncs stops